alacritty_terminal/src/term/mod.rs

```diff
@@ -282,18 +282,6 @@ impl<'a, C> RenderableCellsIter<'a, C> {
                 // Check line-wrapping, leading spacer.
                 || (self.grid[buffer_prev].flags.contains(Flags::LEADING_WIDE_CHAR_SPACER)
                     && selection.contains(prev.col, prev.line))
-        } else if cell.flags.contains(Flags::WIDE_CHAR_SPACER) {
```

Comment:
Does this work if only the wide char spacer is selected? I thought this might be the part we might need to keep, since we still need to mark the wide char cell when the spacer is selected.

Reply:
it's actually what it does, if spacer is selected it selects wide char, the branch I've removed was when wide char is selected -> select spacer, which should automatically work anyway. If you've reached this `if` it means that your wide char isn't selected, thus you check trailing spacer to select a wide char, so everything should be kept this way.